docs/podman-healthcheck-run.1.md

```diff
@@ -0,0 +1,39 @@
+% podman-healthcheck-run(1)
+
+## NAME
+podman\-healthcheck\-run- Run a container healthcheck
+
+## SYNOPSIS
+**podman health run** [*options*]
+
+## DESCRIPTION
+
+Runs the healthcheck command defined in a running container manually.  The resulting error codes are defined
+as follows:
```

Comment:
Would be nice to have an example command like the Docker doc does.

Reply:
oops, more applicable to the next md page.

Reply:
going to leave as-is for now, more will be coming and it will make sense then